en/components/texthighlight.md

```diff
@@ -302,36 +302,14 @@ All the rest of the code in the .ts file is identical to the single container ex
 
 <div class="divider"></div>
 
-### API Summary
+### API and Style References
 
-The following tables summarize some of the useful TextHighlight directive inputs and methods.
+For more detailed information regarding the TextHighlight directive's API, refer to the following link:
+* [`IgxTextHighlight API`]({environment:angularApiUrl}/classes/igxtexthighlightdirective.html)
 
-#### Inputs
-
-The following inputs are available in the **igxTextHighlight** directive:
-
-| Name | Type | Description |
-| :--- | :--- | :--- |
-| `groupName` | string | Defines the group in which the active highlight will be shared. |
-| `row` | number | Defines the row on which the directive is positioned. |
-| `column` | number | Defines the column on which the directive is positioned. |
-| `page` | number | Defines the page on which the directive is positioned. |
-| `value` | any | Defines the underlying value on which we will perform searches. |
-
-<div class="divider"></div>
-
-#### Methods
-
-The following methods are available in the **igxTextHighlight** directive.
-
-| Name | Return type | Parameters | Description |
-| :--- | :--- | :--- | :--- |
-| `highlight` | number | The text to highlight and optionally whether the search is case sensitive and should we highlight only exact matches | Highlights the searched string (if it exists in the value input) and returns how many times the value contains the searched highlighted string|
-| `clearHighlight` | void | none | Clears all existing highlights |
-| `activateIfNecessary` | void | none | Activates a highlight if it is the current active highlight |
-| `setActiveHighlight` | void | The groupName and an object implementing an interface with the row, column and page to activate | Activates the highlight from the specified group on the given row, column and page |
-| `clearActiveHighlight` | void | The group name | Clears the active highlight from the specified group |
+Additional components that were used:
 
+* [`IgxInputGroupComponent`]({environment:angularApiUrl}/classes/igxinputgroupcomponent.html)
```

Comment:
You could provide SASS doc reference for the IgxInputGroupComponent Theme like this :
[`IgxInputGroupComponent Styles`]({environment:sassApiUrl}/index.html#function-igx-input-group-theme